setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='aloception',
+    author='Visual Behavior',
+    version='0.3.0',
+    description='Aloception is a set of package for computer vision: aloscene, alodataset, alonet.',
+    packages=find_packages(include=['aloscene', 'alodataset', 'alonet']),
+    url='https://github.com/Visual-Behavior/aloception',
+    install_requires=[
+        'numpy',
+        'opencv-python',
+        'torch',
+        'torchvision',
+        'matplotlib',
+        'Pillow',
+        'more-itertools',
+        'tensorflow',
+        'pytorch-lightning',
+        'pycocotools',
+        'click',
+        'nvidia-tensorrt',
+        'onnx',
+        'onnxsim',
+        'pycuda',
+        'wandb'],
```

Comment:
This package list is valid for aloception to work. However we get a lot of warning form torch because the version we currently use is an older one.

You should read the file `requirements.txt` in order to get all the needed package and their version.
Maybe you will encounter a problem with numpy. If it's the case you should add `numpy==1.22.2` to `requirements.txt` (this is the version used on the server). I made a similar patch for github action.

The last problem is about pytorch. This package should be installed by the user ([documentation](https://visual-behavior.github.io/aloception/getting_started/getting_started.html#install)) but he doesn't know which version.
**Maybe** it's possible to make the user choose the version he want (cuda, cpu...). If needed you can find all the version number of torch related package on the github action file: `.github/workflows/action.yml`.

Reply:
Along the way, we can also add the required version in the install section in the README.